include/functions.h

```diff
@@ -440,9 +440,9 @@ void func_8002F580(Actor* actor, GlobalContext* globalCtx);
 u32 Actor_HasNoParent(Actor* actor, GlobalContext* globalCtx);
 void func_8002F5C4(Actor* actorA, Actor* actorB, GlobalContext* globalCtx);
 void func_8002F5F0(Actor* actor, GlobalContext* globalCtx);
-u32 Actor_HasChild(GlobalContext* globalCtx, Actor* actor);
-u32 func_8002F63C(GlobalContext* globalCtx, Actor* horse, s32 arg2);
-u32 Actor_HasNoChild(GlobalContext* globalCtx, Actor* actor);
+s32 Actor_IsMounted(GlobalContext* globalCtx, Actor* horse);
+u32 Actor_SetRideActor(GlobalContext* globalCtx, Actor* horse, s32 arg2);
+s32 Actor_NotMounted(GlobalContext* globalCtx, Actor* horse);
```

Comment:
although it is true that the only uses of these functions are in the context of a horse, they are still general purpose checks for the actor having a `child` pointer or not. The same exists for `parent` and is used in many contexts

Reply:
the only thing that tips this off to being intended for horses only is its placement in z_actor next to `Actor_SetRideActor`. still leaning toward a general check currently, though would like to hear others thoughts

Reply:
> the only thing that tips this off to being intended for horses only is its placement in z_actor next to `Actor_SetRideActor`

I think since these functions are only used once in z_en_horse.c (from grepping, but might have missed somewhere) it's also a good indicator
if they were originally intended to be general purpose then I think they would have been used in at least a few places besides horse, but everywhere else just checks directly with like `child == NULL`

but yeah I don't mind changing it back if it fits more with the functions that check for `parent`

Reply:
Yea no, youre right that its only used in horse functions. But yea the existence of both for parent makes it feel like there is an identical set of functions for child, and it just didnt end up getting used for much other than that.
But, the Set ride actor being sandwiched between the two is a good point toward it being horse specific. 

Dont change it yet, and we can see how others feel